src/strategies.rs

```diff
@@ -132,9 +140,10 @@ mod test {
             .with_seed(0)
             .with_max(Duration::from_secs(2))
             .into_iter();
-        for _ in 1..2 {
+        let expected_values = [1.0, 2.0, 2.0];
```

Comment:
can you maybe extrapolate out maybe like 5 values, with the 2 maybe not occurring right away?